spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/okhttp/OkHttpLoadBalancingClient.java

```diff
@@ -85,12 +83,13 @@ public OkHttpRibbonResponse execute(OkHttpRibbonRequest ribbonRequest,
 
 	OkHttpClient getOkHttpClient(IClientConfig configOverride, boolean secure) {
 		OkHttpClient.Builder builder = this.delegate.newBuilder();
-		if (configOverride != null) {
-			builder.connectTimeout(configOverride.get(
+		IClientConfig config = configOverride != null ? configOverride : this.config;
+		if (config != null) {
```

Comment:
Wondering if config can ever be `null` now?  It doesn't appear to be the case now, I think we can get rid of this `if/else`